### ISSUE TITLE
Include source in Core.App and UAP packages

### ISSUE THREAD

**Issue by ericstj** [dotnet/core-setup#2156]
We are flowing symbols and source from corefx but it looks like its not all making it into the meta-packages.

**Comment by ericstj** [comment]
I'm currently blocked on core-setup infrastructure not handling long paths ☹️

**Comment by eerhardt** [comment]
@ericstj - can you explain the user scenario that this is solving?

**Comment by ericstj** [comment]
Including source in the symbols packages.  We publish symbol packages to myget.org and nuget.org.  Without this change those packages are missing the corefx source.

Previously this was all set up for the individual packages by @dagood, and was something I broke when I merged things into a flat package.

I'm not sure we've ever tested end-to-end using this functionality of NuGet.  @dagood may have.

**Comment by dagood** [comment]
I tried it out way back when I first did that for CoreFX, and it worked. (MyGet's symbol source server.) There are other plans to actually index the PDBs to point to GitHub, but it is neat to see the MyGet flow in action.

Note that we don't publish symbol packages to nuget.org--and actually, when you run a NuGet push on symbol packages, they're actually published to symbolsource.org. We stopped doing that for RTM builds for the same reason we don't publish them to MyGet.
